src/encoded/helpers/helper.py

```diff
@@ -1,5 +1,9 @@
 from encoded.vis_defines import vis_format_url
 
+BATCH_DOWNLOAD_DOC_TYPES = [
```

Comment:
I strongly dislike using helper.py ... for anything.    I feel like it was created as a shim for refactoring news.py and region_search.py.     Should batch_download.py be a subclass of a searchView?

Reply:
We can have a discussion on the merits of helper/util functions at some point if so desired. But for now, refactoring the code to eliminate the need for the helper seems tangential to solving this ticket. Still, is it very important to you to address it now?

Reply:
I assume this does not need further action at this time; but is technical debt in our virtual technical debt bucket, on the back of the teal-colored huge elephant in the office :)

Reply:
I defer to @caseylitton on that.